### PR TITLE
SMP-1362 Add monitoring for databases, harness services in helm-charts

### PR DESCRIPTION
**Pull request by sarthak-harnessio**

### CODE REVIEW

File: src/harness/values.yaml [hunk 1]
```diff
@@ -184,6 +184,12 @@ sto:
 # -- overrides for Postgresql
 infra:
```
Comment (arya-harness) [inline src/harness/values.yaml:185]: dont you need to add global fields for values.yaml too?

Comment (arya-harness) [reply src/harness/values.yaml:185]: all do you need to edit the override-demo and override-prod files?

Comment (sarthak-harnessio) [reply src/harness/values.yaml:185]: > all do you need to edit the override-demo and override-prod files?

No, these changes go to values.yaml itself- to enable metrics for harness services. 
I'll make the changes.

**Review by sarthak-harnessio — COMMENTED** [review]
typo error resolved

**Comment by sarthak-harnessio** [comment]
Already merged with release 0.6.0

**Pull request closed without merging**